src/test/java/seedu/parser/ParserTest.java

```diff
@@ -0,0 +1,154 @@
+package seedu.parser;
+
+import org.junit.jupiter.api.Test;
+import seedu.commands.Command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
```

Comment:
Nice work creating the test, has very detailed and wide coverage, could consider renaming ParserTest to MainInputParserTest for standardization.